apps/demos/Demos/DropDownButton/Overview/Vue/App.vue

```diff
@@ -8,31 +8,44 @@
         </div>
         <div class="dx-field-value">
           <DxDropDownButton
-            :items="data.downloads"
-            :drop-down-options="{ width: 230 }"
-            text="Download Trial"
-            icon="save"
-            @item-click="onItemClick"
+              :items="data.downloads"
+              :drop-down-options="{ width: 230 }"
+              text="Download Trial"
+              icon="save"
+              @item-click="onItemClick"
           />
         </div>
       </div>
 
       <div class="dx-field">
-        <div class="dx-field-label">
-          Custom main button action
-        </div>
+        <div class="dx-field-label">Custom main button action</div>
         <div class="dx-field-value">
           <DxDropDownButton
-            :split-button="true"
-            :use-select-mode="false"
-            :items="data.profileSettings"
-            text="Sandra Johnson"
-            icon="../../../../images/gym/coach-woman.png"
-            display-expr="name"
-            key-expr="id"
-            @button-click="onButtonClick"
-            @item-click="onItemClick"
-          />
+              :split-button="true"
+              :use-select-mode="false"
+              :items="data.profileSettings"
```

Comment:
Please don't add unnecessary changes. it is harder to review the PR. If you'd like to make changes to the indentations, it's better to discuss it and do it in another PR